com.ibm.wala.core/src/test/java/com/ibm/wala/core/tests/nestmates/NestmatesTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2006 IBM Corporation.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ */
+package com.ibm.wala.core.tests.nestmates;
+
+import com.ibm.wala.classLoader.IClass;
+import com.ibm.wala.classLoader.IMethod;
+import com.ibm.wala.core.tests.callGraph.CallGraphTestUtil;
+import com.ibm.wala.core.tests.util.WalaTestCase;
+import com.ibm.wala.ipa.callgraph.*;
+import com.ibm.wala.ipa.cha.ClassHierarchy;
+import com.ibm.wala.ipa.cha.ClassHierarchyException;
+import com.ibm.wala.ipa.cha.ClassHierarchyFactory;
+import com.ibm.wala.types.ClassLoaderReference;
+import com.ibm.wala.types.MethodReference;
+import com.ibm.wala.types.TypeReference;
+import com.ibm.wala.util.CancelException;
+import java.io.IOException;
+import java.util.Collection;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class NestmatesTest extends WalaTestCase {
+  @Test
+  public void testPrivateInterfaceMethods()
+      throws ClassHierarchyException, IllegalArgumentException, CancelException, IOException {
+    AnalysisScope scope =
+        CallGraphTestUtil.makeJ2SEAnalysisScope(
+            "wala.testdata11.txt", CallGraphTestUtil.REGRESSION_EXCLUSIONS);
+    ClassHierarchy cha = ClassHierarchyFactory.make(scope);
+    Iterable<Entrypoint> entrypoints =
+        com.ibm.wala.ipa.callgraph.impl.Util.makeMainEntrypoints(
+            scope, cha, "Lnestmates/TestNestmates");
+
+    AnalysisOptions options = CallGraphTestUtil.makeAnalysisOptions(scope, entrypoints);
+
+    CallGraph cg =
+        CallGraphTestUtil.buildZeroCFA(options, new AnalysisCacheImpl(), cha, scope, false);
+
+    // Find node corresponding to main
+    TypeReference tm =
+        TypeReference.findOrCreate(ClassLoaderReference.Application, "Lnestmates/TestNestmates");
+    MethodReference mm = MethodReference.findOrCreate(tm, "main", "([Ljava/lang/String;)V");
+    Assert.assertTrue("expect main node", cg.getNodes(mm).iterator().hasNext());
+    CGNode mnode = cg.getNodes(mm).iterator().next();
+
+    // should be from main to Triple()
+    TypeReference t1s =
+        TypeReference.findOrCreate(ClassLoaderReference.Application, "Lnestmates/Outer$Inner");
+    MethodReference t1m = MethodReference.findOrCreate(t1s, "triple", "()I");
+    Assert.assertTrue("expect Outer.Inner.triple node", cg.getNodes(t1m).iterator().hasNext());
+    CGNode t2node = cg.getNodes(t1m).iterator().next();
```

Comment:
```suggestion
    CGNode t1node = cg.getNodes(t1m).iterator().next();
```
Rename for consistency